core/src/utils/test/playwright/generator.ts

```diff
@@ -8,15 +10,16 @@ export type Direction = 'ltr' | 'rtl';
  * - `high-contrast`: The high contrast light theme values.
  * - `high-contrast-dark`: The high contrast dark theme values.
  */
-export type Theme = 'light' | 'dark' | 'high-contrast' | 'high-contrast-dark';
+export type ThemeMode = 'light' | 'dark' | 'high-contrast' | 'high-contrast-dark';
 
 export type TitleFn = (title: string) => string;
 export type ScreenshotFn = (fileName: string) => string;
 
 export interface TestConfig {
-  mode: Mode;
   direction: Direction;
+  themeMode: ThemeMode;
```

Comment:
I think our whole language model is unsalvageable with our desired goal. This was just to unblock the testing infra to use `theme` for what we are calling themes. 

I'm planning on having a session with the team to align on short term and long term language. I like the idea of `colorScheme`, but we should try to get this language organized before v8 ships... since we are shipping a lot of feature scope under something we are calling "themes" (light, dark, high contrast).

Reply:
We changed it to `palette` in separate PRs, so this is good to go 👍